src/main/scala/org/scalajs/tools/tsimporter/Importer.scala

```diff
@@ -268,6 +268,9 @@ class Importer(val output: java.io.PrintWriter) {
           case TypeName("Array") => QualifiedName.Array
           case TypeName("Function") => QualifiedName.FunctionBase
           case TypeName("object") => QualifiedName.Object
+          case TypeName("PromiseLike") => QualifiedName.Thenable
+          case TypeName("ReadonlyArray") => QualifiedName.JsArray
+          case TypeName("Uint32Array") => QualifiedName.Uint32Array
```

Comment:
If you handle `Uint32Array`, you should definitely handle all the other types that are in `js.typedarray._`.